code/modules/reagents/chemistry_machinery/acid_harness.dm

```diff
@@ -39,13 +39,25 @@
 
 /obj/item/storage/internal/accessory/black_vest/acid_harness
 	storage_slots = 2
+	can_hold = list(
+		/obj/item/reagent_container/glass/beaker,
+		/obj/item/cell,
+	)
+
+/obj/item/storage/internal/accessory/black_vest/acid_harness/can_be_inserted(obj/item/object)
+	. = ..()
+	if(istype(object, /obj/item/reagent_container/glass/beaker))
+		var/obj/item/reagent_container/glass/beaker/possible_beaker = object
+		if(possible_beaker.volume > 60)
+			to_chat(usr, SPAN_NOTICE("[possible_beaker] is too big for the [src]."))
```

Comment:
This should result in a message like: "The beaker is too big for the **the** harness."
Either remove "the" or make it "\the"